### ISSUE TITLE
Issue with Bank Tabs

### ISSUE THREAD

**Issue by Seeker1437** [Hyddwn/HyddwnServer#1]
I have been getting this error when a user is trying to log into a character on their account.

Background information: 

A single account has two characters with the same name on differing servers:

Account: Test1234
    Character: Test1234 [Server: TestServer]
    Character: Test1234 [Server: MainServer]

```
2017-07-04 02:19 [Error] - Problem while loading character '4503599627370500'.
2017-07-04 02:19 [Exception] - System.InvalidOperationException: Bank tab Test1234 already exists.
   at Aura.Channel.World.Inventory.BankInventory.AddTab(String name, Int64 creatureId, BankTabRace race, Int32 width, Int32 height) in c:\Users\Administrator\Desktop\TestServer\aura\src\ChannelServer\World\Inventory\BankInventory.cs:line 47
   at Aura.Channel.Database.ChannelDb.AddBankTabForCharacter(Account account, PlayerCreature character, Int32 width, Int32 height) in c:\Users\Administrator\Desktop\TestServer\aura\src\ChannelServer\Database\ChannelDb.cs:line 379
   at Aura.Channel.Database.ChannelDb.GetCharacter[TCreature](Account account, Int64 entityId, String table) in c:\Users\Administrator\Desktop\TestServer\aura\src\ChannelServer\Database\ChannelDb.cs:line 310
   at Aura.Channel.Database.ChannelDb.GetAccount(String accountId) in c:\Users\Administrator\Desktop\TestServer\aura\src\ChannelServer\Database\ChannelDb.cs:line 101
2017-07-04 02:19 [Debug] - Client 'XXX.XXX.XXX.XXX:62016' : Account <NULL> (Controlling <NULL>) just committed a Severe offense. Incident report: ChannelLogin: Account doesn't contain character 0x10000000000004
```

I think that the way the tabs are stored and accessed have to be changed in order to account for this possibility.

**Comment by Seeker1437** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Resolve Issue #1](https://trello.com/c/gGKskLve/63-resolve-issue-1)

**Comment by Seeker1437** [comment]
Fixed with #2
